bogenliga/src/app/modules/ligatabelle/services/ligatabelle-data-provider.service.ts

```diff
@@ -88,19 +88,79 @@ export class LigatabelleDataProviderService extends DataProviderService {
     });
   }
 
-  //TODO Update funktion hier her mit get arbeiten
-  /*public updateLigatabelleVeranstaltung(id: string | number): Promise<BogenligaResponse<LigatabelleErgebnisDO[]>>{
+  public getLigatabelleWK(id: string | number): Promise<BogenligaResponse<LigatabelleErgebnisDO[]>> {
+    return new Promise((resolve, reject) => {
+      db.ligaTabelle.where('veranstaltungName').equals(id).toArray()
+        .then((data: OfflineLigatabelle[]) => {
+          resolve({result: RequestResult.SUCCESS, payload: fromOfflineLigatabelleArray(data)});
+        }, () => {
+          reject({result: RequestResult.FAILURE});
+        });
+    });
+  };
+
+  public async updateMannschaftLT(id : number, satzpunkte:number, satzpunkteGegner : number, spd :number, matchpunkte : number, matchpunkteGegner : number){
```

Comment:
updateMannschaftLT und updateLigatabelleVeranstaltung sollte man dann in der Zukunft auch eher in schusszettel.component.ts moven, weil die eigentlich nicht so ganz in eine data-provider datei passen